### ISSUE TITLE
[Bug] api.blocks.render empty blocks cause "Cannot read property 'holder' of undefined"

### ISSUE THREAD

**Issue by cdllqos** [codex-team/editor.js#1740]
Describe a bug.

Steps to reproduce:
1. When editor is prepared
2. Call `api.blocks.render({blocks:[]})`
3. Click editor，throw `Cannot read property 'holder' of undefined`

Expected behavior:  editor works no error

Screenshots:
![image](https://user-images.githubusercontent.com/25027791/128332412-c5864547-3903-4f7a-9956-94025ab37644.png)

Device, Browser, OS:

Editor.js version: 2.22.2

Plugins you use with their versions: no plugins

**Comment by christopher-caldwell** [comment]
YES! I thought I was insane, the same thing is happening to me with v2.22.2

**Comment by anstapol** [comment]
Same here...

**Comment by eyalcohen4** [comment]
Same here, does someone found a workaround? Need to access the holder of a tool
